src/clj/rems/form_validation.clj

```diff
@@ -26,14 +27,21 @@
   (and (= (:field/type field) :option)
        (not (option-value-valid? field))))
 
+;; TODO: validate that attachments are actually valid?
+(defn- invalid-attachment-value? [field]
```

Comment:
How about renaming this method so that it doesn't get forgotten to the comment that there is no "real" validation to attachments? Sth like validate-attachment-id?